src/types/bridge-events.d.ts

```diff
@@ -88,7 +88,12 @@ export type HostToGuestEvent =
   /**
    * The host informs guests that the sidebar layout has been changed.
    */
-  | 'sidebarLayoutChanged';
+  | 'sidebarLayoutChanged'
+
+  /**
+   * The host informs guests to scroll to the closest annotation from the set.
+   */
+  | 'scrollToAnnotations';
```

Comment:
This name is very similar to `sidebar-host` event, `scrollToAnnotation`. On acts on a single annotation tag, but this new event acts on a list of annotation tags.

I am open to alternative names: `scrollToClosestAnnotation` for example.